ocp_resources/utils.py

```diff
@@ -276,3 +276,39 @@ def _return_resource(_resource, _check_exists, _msg):
                     _check_exists=check_exists,
                     _msg=skip_create_warn_msg,
                 )
+
+
+def add_rules(role_object):
+    """
+    Add a list of rules to a Role / CluterRole object.
+
+    Args:
+        role_object (list): list of dicts of rules. In the dict:
+                permissions_to_resources (list): List of string with resource/s to which you want to add permissions to.
+                Verbs (list): Determine the action/s (permissions) applicable on a specific resource.
+                    Available verbs per resource can be seen with the command 'oc api-resources --sort-by name -o wide'
+    """
+
+    def _add_rule(
```

Comment:
The inner function is not needed, 
can be called in the for loop

Reply:
Ack.